site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties

```diff
@@ -423,7 +423,7 @@ sinfo.list.desc.ellipse=...
 sinfo.list.desc.more=(More)
 sinfo.language = Site Language
 sinfo.language.defaultUserLanguage = User Language (default)
-
+sinfo.manage.participants.moved = Manage Participants (previously here) has been moved to a new page on the <a href="{0}">Manage Participants tab</a>.
```

Comment:
How about:
`The Participant List (previously located here) has moved to its own page called <a href="{0}">Manage Participants</a>, linked in the tabs above.`
??